### ISSUE TITLE
L.control.browserPrint - Custom print

### ISSUE THREAD

**Issue by EricCronje** [Igor-Vladyka/leaflet.browser.print#85]
Thanks for the plug in. 
We have a map with circles drawn on it. When selecting a map with alot (+_900) of circles and the map scaled to center. Then we also want a summary table or custom content to show on the bottom. When selecting the custom option - to print. then the image and table is not filling the whole page (A4) portrait.

Screen shot below (With alot of circles)

![image](https://user-images.githubusercontent.com/34335457/75325157-02401100-5881-11ea-8820-a8fe3eb56245.png)

Should look like this:

![image](https://user-images.githubusercontent.com/34335457/75326270-f5bcb800-5882-11ea-9f1f-6ef575ecf82a.png)

**Expected behavior**
The idea is that it fills the page. All must fit on one page.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome

Using leaflet version: 1.4.0
And referencing: leaflet.browser.print

Code:

            L.control.browserPrint({
                title: thisBlock.name,
                printModes: ["Custom"]
            }).addTo(map)

And then marked the summary table:

<div class="holeSummaryTableSceleton sub-content" leaflet-browser-print-content>
 ...Content
 </div>

**Comment by Igor-Vladyka** [comment]
Hi.

You would need to style content something like
```
        .grid-print-container { // grid holder that holds all content (map and any other content)
		grid-template: 1fr auto / 1fr;
	}
	.grid-map-print { // map container itself
		grid-row: 1;
	}
	.table{
		grid-row: 2;
	}
```

Please check [Printing additional content section](https://github.com/Igor-Vladyka/leaflet.browser.print#printing-additional-content-section) for more details and let me know. If it still not working, please specify more details and/or create a demo version to debug it properly.

Regards,
Igor

**Comment by Igor-Vladyka** [comment]
@EricCronje  Have you managed to achive what you where looking for?

**Comment by smyli323** [comment]
Hi @Igor-Vladyka,

I am having a similar problem with the plug-in.  When I try to print in portrait mode, the map just doesn't fill the page.  I've compared the css & html from one of your examples with my angular application and everything looks the same as far as I can tell.  I have tried adding the css as you suggested above for .grid-print-container & .grid-map-print but no luck.  Any help would be greatly appreciated!

![image](https://user-images.githubusercontent.com/9968667/139965515-f685cd74-3242-4c2a-9502-ac2ab9644c01.png)
